### DISCUSSION TITLE
javax.json.bind.JsonbException

### DISCUSSION THREAD

**Discussion by landreu23** [quarkusio/quarkus#32329] [Community]
buen dia presto este error cuando intento hacer una pruba con el postman     
    "details": "Error id 82ed4b77-deeb-469c-9ca8-7422d3658121-13, javax.json.bind.JsonbException: Unable to serialize property 'entityClass' from org.jboss.resteasy.specimpl.BuiltResponse",

**Comment by maxandersen** [comment]
Hi,

Sorry to hear you are having that issue.

To help it will be best if you write your question in English and provide some more details - as a "I get this error in postman" does not give us much to go from.

Thank you.
